strawberry/graphql.py

```diff
@@ -38,14 +39,17 @@
     if validation_errors:
         return ExecutionResult(data=None, errors=validation_errors)
 
-    return graphql_excute(
+    result = graphql_excute(
         schema,
         parse(query),
         middleware=[DirectivesMiddleware()],
         variable_values=variable_values,
         operation_name=operation_name,
         context_value=context_value,
     )
+    if isawaitable(result):
+        result = await typing.cast(typing.Awaitable[ExecutionResult], result)
```

Comment:
Not sure if this is the best way to convince MyPy that `result` is of the correct type. I haven't used static typing in any of my Python projects so far.

Reply:
I think this is the right way for doing so!